six/modules/c++/six.sicd/include/six/sicd/Utilities.h

```diff
@@ -337,6 +337,49 @@ class Utilities
      * \return mock ComplexData object
      */
     static std::auto_ptr<ComplexData> createFakeComplexData();
+
+    /*!
+     * If data has no areaPlane, derive it and set it
+     * \param data ComplexData to populate with areaPlane
+     * \param includeSegmentList Should the areaPlane's segmentlist be filled?
+     */
+    static void setAreaPlane(
+            six::sicd::ComplexData& data,
+            bool includeSegmentList=true);
+
+    /*!
+     * Just fill in an AreaPlane separate from the ComplexData
```

Comment:
I think you can spell this out a bit more in the comments.  Something like "Derives an output plane definition from existing SICD metadata"